springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/annotation/AsyncAnnotationScannerUtil.java

```diff
@@ -79,4 +81,23 @@ public static Map<String, MessageBinding> processMessageBindingFromAnnotation(Me
                 .map(Optional::get)
                 .collect(Collectors.toMap(ProcessedMessageBinding::getType, ProcessedMessageBinding::getBinding));
     }
+
+    public static Message processMessageFromAnnotation(Method method) {
```

Comment:
Can we add two tests that check this methods with and without annotation

Reply:
Done. Also added the missing `processMessageBindingFromAnnotation` test